test/framework/options.py

```diff
@@ -1898,6 +1898,13 @@ def test_review_pr(self):
         self.mock_stdout(False)
         self.assertTrue(re.search(r"^Comparing zlib-1.2.8\S* with zlib-1.2.8", txt))
 
+    def test_allow_values_that_are_probably_meant_to_be_options(self):
+        """Test detecting typos or ambiguities on the eb command line."""
+        common_args = []
+
+        eb_go = parse_options(args=common_args + ['-r', 'f'])
+        self.assertFalse(eb_go.options.robot == 'f')
```

Comment:
@stdweird: for same reason, this doesn't work... `-r f` should be rejected by generaloption because there's an option `-f`, but it's not

I checked a bit, and for some reason `is_value_a_commandline_option` is getting `()` as a value passed in, rather than `f` which would lead to the reject

any idea? `--robot`/`-r` is a `store_or_None` option, could that be the reason?